examples.md

```diff
@@ -108,3 +109,13 @@ uses: actions/cache@preview
     restore-keys: |
       ${{ runner.os }}-go-
 ```
+
+## Elixir - Mix
+```yaml
+- uses: actions.cache@preview
```

Comment:
Should be `uses: actions/cache@preview` (`/` instead of `.`)

Reply:
Sorry, typo😫

Reply:
@joshmgross  Fixed it.